### DISCUSSION TITLE
rebar stopped working after half a year working good

### DISCUSSION THREAD

**Discussion by fab98bc** [xCuri0/ReBarUEFI#267] [General]
**System**
 - Motherboard: asus maximus VIII gene
 - BIOS Version: 3801
 - GPU: RTX 3070 ROG STRIX
 - [ ] CSM is turned OFF
 - [ ] 4G decoding is enabled. 
 - [ ] UEFIPatch is applied
 - [ ] I have read [Common issues (and fixes)](https://github.com/xCuri0/ReBarUEFI/wiki/Common-issues-(and-fixes))

**Description**
after half a year running flawlessly, today just stopped working and I dont know why. wake from sleep always work without single issue but today it fails error code E3 o wake. 

then I set rebar size 32 as always and it wont post. tried another bios chip with a fresh install via epromm programer, same issue. after activate 4g decoding and desactivate cms. rebar set to 32 it wont boot. keeps in a0 state with white vga led on. if I set size to 0 to desactivate it, everything works fine. 

all of this never happened before. It always worked good .

only new thing in my pc is win11 update (KB5040527). 

Please help :(

**Comment by xCuri0** [comment]
Smaller sizes work ? (like 10)

**Comment by fab98bc** [comment]
> Smaller sizes work ? (like 10)

No it doesn't work. Now stuck at error code 99 while booting. Orange "boot device led" on.

![20240801_103337](https://github.com/user-attachments/assets/c41b97f3-9017-4888-acd8-6a599b5dadb3)

**Comment by xCuri0** [comment]
@fab98bc did you verify if GPU-Z shows that it's running at x16 Gen3 (under load otherwise it will show Gen1.1), reseating the GPU might fix if it's some issue like that.

**Comment by fab98bc** [comment]
> @fab98bc did you verify if GPU-Z shows that it's running at x16 Gen3 (under load otherwise it will show Gen1.1), reseating the GPU might fix if it's some issue like that.


I have a pci nvme adapter so my gpu runs at 8x 3.0 But I never had a single issue there and since I haven't done any modifications, software or hardware related is very strange that it stopped working...

I tried removing the adapter but got the same result and yes, GPU is running 16x 3.0.

Testing different things, If I enable iGPU it boots giving display output through dGPU but it doesn't give me enter bios screen just boots right into windows and wake from sleep works. But it keeps VGA check led on in motherboard.

I removed and plugged in again the GPU, reinstalled driver using DDU, updated VBIOS again. But nothing seems to work.

Another thing worth to mention is that I'm using an Intel i9 9980hk CPU bga to lga converted... bios is modified to make it work and with rebar mod applied. It's been working perfect all this time so far. Until now :(

**Comment by fab98bc** [comment]
Still nothing works. After enabled rebar pc boots with no image. Using remote desktop i can see that it boot but seems the gpu doesn't start correctly. After disable rebar it works normally but I noticed csm enables itself again on bios. All my drives are gpt partitioned

**Comment by xCuri0** [comment]
@fab98bc does changing the PCIe generation in the BIOS make any difference ?

Try setting it to Gen2, Resizable BAR should still work with Gen2.
